dwave/cloud/utils.py

```diff
@@ -546,15 +546,15 @@ def _argshash(self, args: List[Any], kwargs: Dict[Any, Any]):
 
     def __init__(self, *,
                  maxage: Optional[float] = None,
-                 cache: Optional[Mapping] = None,
```

Comment:
Do we want to still support `cache` with a deprecation warning?

Probably too low level for it to matter I suppose.

Reply:
I expect this to have ~zero impact on users. It's not just semi-private (undocumented), but also you don't need to set `cache` param except for testing or exotic use cases...